src/compiler/Restler.Compiler.Test/ExampleTests.fs

```diff
@@ -44,7 +44,7 @@ module Examples =
 
         [<Fact>]
         let ``array example in grammar without dependencies`` () =
-            let grammarDirectoryPath = ctx.testRootDirPath
+            let grammarDirectoryPath = @"c:\temp\restler" //ctx.testRootDirPath
```

Comment:
why is ctx.testRootDirPath now commented out? (is this left-over ad-hoc test code?)

Reply:
Thanks for catching, fixed.